docs/jitos/spec-0000.md

```
@@ -0,0 +1,383 @@
+(previous_graph_state, rewrite_txn) -> (new_graph_state)
+```
+
+The kernel interprets this rewrite log as the _causal history_ of the OS.
+
+---
+
+## 2. What This Demo Proves
+
+This first demo page proves 5 concepts interactively:
+
+1. **Immutable append-only rewrites**    
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 4

(MD009, no-trailing-spaces)